package.json

```diff
@@ -14,9 +14,9 @@
     "test:e2e:node-module": "ts-mocha -p tsconfig.json --recursive 'test/e2e/*.test*' --timeout 120000",
     "test:e2e:cli": "sh ./test/e2e/e2e-cli.sh",
     "test:coverage": "nyc npm run test; nyc report --reporter=lcov",
-    "build": "rimraf ./lib && npx tsc",
+    "build": "rm -rf ./lib && npx tsc",
     "dev": "npx tsc --watch",
-    "prepare": "npm run build",
+    "prepublishOnly": "npm run build",
```

Comment:
The `prepare` script here will build the package before publishing and after `npm install`. This is a problem for this script because it will try to rebuild after `npm ci --omit=dev`, which will always end up with it failing because it requires dev dependencies to build. 

By changing this to `prepublishOnly`, we're able to eliminate implicit and unnecessary builds from occurring. 

Frankly, this also eliminates an unobvious and implicit step from occurring in our automated processes, so I think this is a positive step forward even if some minor consequences are realized.